sdcm/argus_results.py

```diff
@@ -106,6 +113,12 @@ def send_result_to_argus(argus_client: ArgusClient, workload: str, name: str, de
                                     row=f"Cycle #{cycle}",
                                     value=value,
                                     status=Status.PASS if value < operation_error_thresholds[f"percentile_{percentile}"] else Status.ERROR)
+        if value := summary[operation.upper()].get("throughput", None):
+            result_table.add_result(column=f"Throughput {operation}",
+                                    row=f"Cycle #{cycle}",
+                                    value=value,
+                                    status=Status.PASS)
```

Comment:
better set `UNSET` if we don't validate it.
(might need to rebase this PR as this status was added just recently)

Reply:
I plan to validate it in gradual test. `PASS` is temporary status. Added `TODO`

Reply:
so UNSET should be the temporary.

@soyacz is working on a better way to express the logic for validation we'll want to do for each metric, and we'll align to that when it would be ready

Reply:
changed